packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -89,9 +101,8 @@ const isRenamingDataset = ref(false);
 const rawContent: Ref<CsvAsset | null> = ref(null);
 const isDatasetLoading = ref(false);
 const selectedTabIndex = ref(0);
-
 const view = ref(DatasetView.DESCRIPTION);
-
+const image: Ref<string | undefined> = ref(undefined);
```

Comment:
fyi you can just do `const image = ref<string | undefined>(undefined);` no need to import `Ref`

Reply:
You can... however every time I work in the front end I find a number of issues relating to types... so I now make a point to obsessively type everything :)